_data/glossary/configmap.yaml

```diff
@@ -0,0 +1,13 @@
+id: configmap
+name: ConfigMap
+full-link: /docs/tasks/configure-pod-container/configmap/
+related:
+- pod
+- secret
+tags:
+- core-object
+short-description: >
+  Used to store non confidential data in key-value pairs and consumed as environment variables, command-line arguments, or config files in a volume
```

Comment:
A few things:
1. non confidential -> non-confidential
2. link volume to the tool tip: 
volume -> {% glossary_tooltip text="volume" term_id="volume" %}

3. Also, add a period to the end of the phrase.

Reply:
done